R/Workflow.R

```diff
@@ -264,8 +264,9 @@ rm(list=ls()[!ls() %in% c("df_raw","date_formats","datetime_formats","trackingda
 ## USER INPUT START ##
 #--------------------#
 
+metadata_path <- glue::glue("{species_code}_Metadata.csv")
```

Comment:
Add dependency on glue here. Could just use a paste0() to remove this dependency

Reply:
I can maybe merge changes and then update this afterwards

Reply:
Updated now